packages/apollo/src/commands/client/__tests__/__snapshots__/generate.test.ts.snap

```diff
@@ -37,69 +37,6 @@ exports[`client:codegen generates operation IDs for swift files when flag is set
 }"
 `;
 
-exports[`client:codegen writes TypeScript global types to a custom path when globalTypesFile is set 1`] = `
```

Comment:
Why was this removed?

Reply:
This was a result of muddying up, then deleting, and finally regenerating this test snap. The test is skipped, so the snap is obsolete. Side note: doesn't jest normally find these and remove them with `-u`?

Can totally revert this change!